entrypoint.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+export DJANGO_SETTINGS_MODULE=embark.settings.docker
+export EMBARK_DEBUG=True
+export HTTP_PORT=80
+export HTTPS_PORT=443
+export BIND_IP='0.0.0.0'
+export FILE_SIZE=2000000000
+
```

Comment:
Should we also define /app/www somewhere. In the future we just need to change it once